product-service/serverless.ts

```diff
@@ -26,6 +26,9 @@ const serverlessConfiguration: Serverless = {
     },
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      PG_HOST: 'nodejs-aws-be.cupqagybo1vs.eu-west-1.rds.amazonaws.com',
```

Comment:
Please use env variables. It is absolutely insecure. It is really sad that our lector did not mentioned about this. But I strongly suggest you to replace this hardcoded values with env variables

Reply:
Sure, at least I decided not to commit credentials. In real-world application a secret vault should be used instead, maybe this topic will be covered in future lections.